Source/LinqToDB/DataProvider/Oracle/Oracle11SqlOptimizer.cs

```diff
@@ -51,7 +50,10 @@ public override bool IsParameterDependedElement(IQueryElement element)
 
 					// Oracle saves empty string as null to database, so we need predicate modification before sending query
 					//
-					if (expr.Operator.In(SqlPredicate.Operator.Equal, SqlPredicate.Operator.NotEqual, SqlPredicate.Operator.GreaterOrEqual, SqlPredicate.Operator.LessOrEqual) && expr.WithNull == true)
+					if ((expr.Operator == SqlPredicate.Operator.Equal          ||
+						 expr.Operator == SqlPredicate.Operator.NotEqual       ||
+						 expr.Operator == SqlPredicate.Operator.GreaterOrEqual ||
+						 expr.Operator == SqlPredicate.Operator.LessOrEqual) && expr.WithNull == true)
```

Comment:
Make the additional condition outside the `()` more obvious by putting it on the next line.
```suggestion
					if ((expr.Operator == SqlPredicate.Operator.Equal          ||
						 expr.Operator == SqlPredicate.Operator.NotEqual       ||
						 expr.Operator == SqlPredicate.Operator.GreaterOrEqual ||
						 expr.Operator == SqlPredicate.Operator.LessOrEqual) && 
					    expr.WithNull == true)
```

Reply:
I'll exchange conditions instead as it looks more performant